tools/site_and_regional/README

```diff
@@ -27,20 +27,8 @@ mknoocnmap.pl
         Script to create unity mapping dataset for single-point
         or regional studies over land-only (no ocean).
 
-getregional_datasets.pl 
-        Extract out regional datasets from global ones and put files in a location that
-        can be used by build-namelist.
-
 NCL Scripts available:
 
 mkunitymap.ncl
         NCL script to create a unity map -- ran by above script (mknoocnmap.pl)
 
```

Comment:
what is this used for ?  is it still needed?

Reply:
mkunitymap.ncl? We still might need it for WRF type regional cases.  But, it's unclear if it's really needed anymore. We need to map exactly how WRF cases should be done. I think that's a different conversation that we need to do.

Reply:
This will be handled with #1513. So resolving this conversation.

Reply:
This is needed for WRF cases. Also this might not be handled in #1513 because from my understanding WRF-cTSM still uses MCT under the hood, which requires domain files. (And this was needed for creating domain files.)